cornichon-kafka/src/main/scala/com/github/agourlay/cornichon/kafka/KafkaStepBuilder.scala

```diff
@@ -0,0 +1,40 @@
+package com.github.agourlay.cornichon.kafka
+
+import com.github.agourlay.cornichon.core.SessionKey
+import com.github.agourlay.cornichon.json.CornichonJson._
+import com.github.agourlay.cornichon.json.JsonPath
+import com.github.agourlay.cornichon.json.JsonSteps.JsonStepBuilder
+import com.github.agourlay.cornichon.matchers.MatcherResolver
+import com.github.agourlay.cornichon.resolver.PlaceholderResolver
+import com.github.agourlay.cornichon.steps.regular.assertStep.{ AssertStep, Assertion, GenericEqualityAssertion }
+import cats.instances.string._
+
+case class KafkaStepBuilder(
+    sessionKey: String,
+    private val placeholderResolver: PlaceholderResolver,
+    private val matcherResolver: MatcherResolver
+) {
+  def topic_is(expected: String) = AssertStep(
+    title = s"topic is $expected",
+    action = s ⇒ Assertion.either {
+      for {
+        messageObject ← s.getJsonStringField(s"$sessionKey-topic")
```

Comment:
need an extra import to make the Either right biased in Scala 2.11
`import cats.instances.either._`
https://travis-ci.org/agourlay/cornichon/jobs/317961356#L653